.github/workflows/check-frame-omni-bencher.yml

```
@@ -77,10 +83,13 @@ jobs:
         run: |
           RUNTIME_BLOB_NAME=$(echo $PACKAGE_NAME | sed 's/-/_/g').compact.compressed.wasm
           RUNTIME_BLOB_PATH=./target/release/wbuild/$PACKAGE_NAME/$RUNTIME_BLOB_NAME
-          forklift cargo build --release --locked -p $PACKAGE_NAME -p frame-omni-bencher --features runtime-benchmarks
+          forklift cargo build --release --locked -p $PACKAGE_NAME -p frame-omni-bencher --features runtime-benchmarks --quiet
```

some of them might require riscv (temporarily) so it's better to read --features also from config
```suggestion
          forklift cargo build --release --locked -p $PACKAGE_NAME -p frame-omni-bencher --features ${{ matrix.runtime.bench_features }} --quiet
```